docs/chaos/result.md

```diff
@@ -0,0 +1,141 @@
+# Outcome monkey strategy
+
+## About
+
+- **Options**:
+  - [`OutcomeStrategyOptions`](xref:Polly.Simmy.Behavior.OutcomeStrategyOptions)
+  - [`OutcomeStrategyOptions<T>`](xref:Polly.Simmy.Behavior.OutcomeStrategyOptions`1)
+- **Extensions**: `AddChaosResult`
+- **Strategy Type**: Reactive
+
+---
+
+The outcome chaos strategy is designed to inject or substitute fake results into system operations. This allows testing how an application behaves when it receives different types of responses, like successful results, errors, or exceptions.
```

Comment:
Is it correct to mention exceptions here? I thought the Fault strategy was for that?

Reply:
Since this strategy works with `Outcome`, it allows returning exceptions too.